### DISCUSSION TITLE
Code-OSS Related Questions.

### DISCUSSION THREAD

**Discussion by atikattar1104** [termux-user-repository/tur#799] [Q&A] [answered]
Hi. I Recently Found Out That VS Code Can Run On Termux, Thanks To TUR Repo. But As Soon As I Heard About It, A Couple Of Questions Popped Up In My Mind. They Are As Follows. 

1. As The Name Of VS Code Is `code-oss` In TUR Repo, Is It Similar To VSCodium, Which Is VS Code But Has It's Telemetry Data/Code Removed? Are These 2 (`code-oss` From TUR Repo And VSCodium) Similar?

2. If They Are Similar, Did TUR Repo's `code-oss` Has It's Telemetry Data/Code Removed (Just Like VSCodium) In The Build Process?

Actually, I Am Currently Learning Web Development In My College. Almost Everyone Have A Laptop, Except Me. I Am Using An Open Source Android IDE Named `ACode`, Which Is A Little Unstable, And Have Some Issues With Heavy Working. That's Why I Am Looking For VS Code In Termux.

Also, I Don't Trust Microsoft When It Comes To Data Privacy. That's Why Asking These Questions. Please Let Me Know About It.

Thanks!

**Comment by licy183** [comment, marked as answer]
`code-oss` in TUR is not like `vscodium`. In `code-oss`, telemetry is not removed, but it is turned off by default. (See [0004-set-enableTelemetry-to-false-by-default.patch](https://github.com/termux-user-repository/tur/blob/master/tur/code-oss/0004-set-enableTelemetry-to-false-by-default.patch))

**Reply by atikattar1104** [reply]
Thanks! Despite Having Dissimilarities With VSCodium, The `code-oss` Seems Promising To Me. I Will Try If I Can Install It In My Potato Phone 😅.

**Comment by atikattar1104** [comment]
I Am Keeping This Discussion Open, In Case Anyone Else Have Questions Related To `code-oss`.

**Comment by ilkersigirci** [comment]
Can the [code-marketplace](https://aur.archlinux.org/packages/code-marketplace) be integrated into Termux so that we can use extensions like `Remote-ssh` and `Copilot`?

**Reply by licy183** [reply]
It is not possible, because these extensions use prebuilt binary files, and these binary files are linked against glibc, not bionic libc, unless these extensions are open-sourced, and TUR recompiles those binary files to let them link against bionic libc.
